src/System.Private.ServiceModel/tools/scripts/SetupWcfIISHostedService.cmd

```diff
@@ -199,6 +202,11 @@ echo.
 echo All operations completed successfully!
 goto :Done
 
+:Unlock
+call :Run %_appcmd% unlock config /section:%1
+if ERRORLEVEL 1 goto :Failure
```

Comment:
Agreed. The redundant line is limited in this case, and it's cleaner because we jump less in the script.